mods/ra/maps/soviet-04a/map.yaml

```diff
@@ -0,0 +1,754 @@
+		Owner: Neutral
+	Actor29: sbag
+		Location: 90,56
+		Owner: Neutral
+	Actor30: sbag
+		Location: 91,56
+		Owner: Neutral
+	Actor31: sbag
+		Location: 92,56
+		Owner: Neutral
+	Actor32: sbag
+		Location: 67,57
+		Owner: Neutral
+	Actor33: sbag
+		Location: 68,57
+		Owner: Neutral
+	Actor34: sbag
+		Location: 69,57
+		Owner: Neutral
+	Actor35: sbag
+		Location: 70,57
+		Owner: Neutral
+	Actor36: sbag
+		Location: 71,57
+		Owner: Neutral
+	Actor37: sbag
+		Location: 72,57
+		Owner: Neutral
+	Actor38: sbag
+		Location: 78,58
+		Owner: Neutral
+	Actor39: sbag
+		Location: 79,58
+		Owner: Neutral
+	Actor40: sbag
+		Location: 80,58
+		Owner: Neutral
+	Actor41: sbag
+		Location: 81,58
+		Owner: Neutral
+	Actor42: sbag
+		Location: 82,58
+		Owner: Neutral
+	Actor43: sbag
+		Location: 83,58
+		Owner: Neutral
+	Actor44: sbag
+		Location: 57,69
+		Owner: Neutral
+	Actor45: sbag
+		Location: 61,69
+		Owner: Neutral
+	Actor46: sbag
+		Location: 44,72
+		Owner: Neutral
+	Actor47: sbag
+		Location: 48,72
+		Owner: Neutral
+	Actor48: tc04
+		Location: 97,67
+		Owner: Neutral
+	Actor49: tc05
+		Location: 71,79
+		Owner: Neutral
+	Actor50: t10
+		Location: 100,59
+		Owner: Neutral
+	Actor51: t13
+		Location: 78,50
+		Owner: Neutral
+	Actor52: tc01
+		Location: 68,55
+		Owner: Neutral
+	Actor53: t08
+		Location: 73,47
+		Owner: Neutral
+	Actor54: tc01
+		Location: 29,86
+		Owner: Neutral
+	Actor55: t08
+		Location: 36,84
+		Owner: Neutral
+	Actor56: t06
+		Location: 36,87
+		Owner: Neutral
+	Actor57: t11
+		Location: 38,74
+		Owner: Neutral
+	Actor58: tc02
+		Location: 42,75
+		Owner: Neutral
+	Actor59: tc03
+		Location: 60,80
+		Owner: Neutral
+	Actor60: tc05
+		Location: 54,73
+		Owner: Neutral
+	Actor61: tc01
+		Location: 47,87
+		Owner: Neutral
+	Actor62: t16
+		Location: 56,79
+		Owner: Neutral
+	Actor63: tc04
+		Location: 79,70
+		Owner: Neutral
+	Actor64: tc01
+		Location: 108,73
+		Owner: Neutral
+	Actor65: t07
+		Location: 94,60
+		Owner: Neutral
+	Actor66: tc05
+		Location: 91,66
+		Owner: Neutral
+	Actor67: t17
+		Location: 82,64
+		Owner: Neutral
+	Actor68: tc04
+		Location: 28,80
+		Owner: Neutral
+	Actor69: tc05
+		Location: 29,75
+		Owner: Neutral
+	Actor70: tc02
+		Location: 27,76
+		Owner: Neutral
+	Actor71: t17
+		Location: 42,52
+		Owner: Neutral
+	Actor72: t01
+		Location: 53,81
+		Owner: Neutral
+	Actor73: tc01
+		Location: 99,77
+		Owner: Neutral
+	Actor74: mine
+		Location: 106,86
+		Owner: Neutral
+	Actor75: mine
+		Location: 70,87
+		Owner: Neutral
+	Actor76: tc04
+		Location: 39,46
+		Owner: Neutral
+	Actor77: tc05
+		Location: 45,61
+		Owner: Neutral
+	Actor78: tc02
+		Location: 41,48
+		Owner: Neutral
+	Actor79: tc04
+		Location: 98,45
+		Owner: Neutral
+	Actor80: t15
+		Location: 95,48
+		Owner: Neutral
+	Actor81: tc03
+		Location: 38,68
+		Owner: Neutral
+	Actor82: t08
+		Location: 38,67
+		Owner: Neutral
+	Actor83: t08
+		Location: 40,69
+		Owner: Neutral
+	Actor84: tc05
+		Location: 38,79
+		Owner: Neutral
+	Silo1: silo
```

Comment:
Please sort these entries so that the named ones are at the end and the generic ones (Actor123 etc.) at the beginning.